js/index.js

```diff
@@ -0,0 +1,325 @@
+    switch (x) {
+        case "lowFirst":
+            productList.sort((a, b) => {
+                return a.price - b.price;
+            });
+            localStorage.setItem('productList', JSON.stringify(productList));
+            showData();
+            break;
+        case "highFirst":
+            productList.sort((a, b) => {
+                return b.price - a.price;
+            });
+            localStorage.setItem('productList', JSON.stringify(productList));
+            showData();
+            break;
+        case "idFirst":
+            productList.sort((a, b) => {
+                return a.productid - b.productid;
+            });
+            localStorage.setItem('productList', JSON.stringify(productList));
+            showData();
+            break;
+        case "nameFirst":
+            productList.sort(function (a, b) {
+                var x = a.productname.toLowerCase();
+                var y = b.productname.toLowerCase();
+                return x < y ? -1 : x > y ? 1 : 0;
+            });
+            localStorage.setItem('productList', JSON.stringify(productList));
+            showData();
+            break;
+    }
+}
+
+//Function to show image while adding or editing
+function readURL(input) {
+    document.getElementById("image").src = "";
+    if (input.files || input.files[0]) {
+        var reader = new FileReader();
+        reader.onload = function (e) {
+            $('#imgedit')
+                .attr('src', e.target.result)
+                .width(80)
+                .height(80);
+        };
+
+        reader.readAsDataURL(input.files[0]);
+    }
+}
+
+//Function to show the input data 
+function showData() {
+    var productList;
+    if (localStorage.getItem("productList") == null) {
+        productList = [];
+    }
+    else {
+        productList = JSON.parse(localStorage.getItem("productList"));
+        var html = "";
+        productList.forEach(function (element, index) {
+            html += "<tr>";
+            html += "<td>" + element.productid + "</td>";
+            html += "<td>" + element.productname + "</td>";
+            html += "<td> <img src=" + element.image + " style='height:80px; width:80px;' alt='Image Product' </img></td>";
+            html += "<td> &#8377;" + element.price + "</td>";
+            html += "<td>" + element.description + "</td>";
+            html += "<td> <button onclick='editData(" + index + ")' data-toggle='modal' data-target='#addAdmin' class='btn btn-success'>Edit</button></td>";
+            html += "<td> <button onclick='deleteData(" + index + ")' class='btn btn-danger'>Delete</button></td>";
+            html += "</tr>";
+        });
+        document.querySelector("#dataTable tbody").innerHTML = html;
+    }
+}
+
+//Shows data while windows load.
+document.onload = showData();
+
+//Function to add data
+function addData() {
+    if (validateForm() == true && validateImage() == true && validateIdName() == true) {
```

Comment:
No need to check **== true**. When scenario with only True or False. 
i.e.  If(validateForm() && validateImage()  && validateIdName()) 
You can do like this.